{
  "Header": {
    "Original": "CVE-2020-0093 (In exif_data_save_data_entry of exif-data.c, there is a possible out o ...)",
    "Line": 231077,
    "ID": "CVE-2020-0093",
    "Description": "(In exif_data_save_data_entry of exif-data.c, there is a possible out o ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2214-1}",
      "Line": 231078,
      "Type": "xref",
      "Bugs": [
        "DLA-2214-1"
      ]
    },
    {
      "Original": "- libexif 0.6.21-8",
      "Line": 231079,
      "Type": "package",
      "Package": "libexif",
      "Kind": "fixed",
      "Version": "0.6.21-8"
    },
    {
      "Original": "[buster] - libexif 0.6.21-5.1+deb10u2",
      "Line": 231080,
      "Type": "package",
      "Release": "buster",
      "Package": "libexif",
      "Kind": "fixed",
      "Version": "0.6.21-5.1+deb10u2"
    },
    {
      "Original": "[stretch] - libexif 0.6.21-2+deb9u2",
      "Line": 231081,
      "Type": "package",
      "Release": "stretch",
      "Package": "libexif",
      "Kind": "fixed",
      "Version": "0.6.21-2+deb9u2"
    },
    {
      "Original": "NOTE: https://github.com/libexif/libexif/issues/42",
      "Line": 231082,
      "Type": "NOTE",
      "Description": "https://github.com/libexif/libexif/issues/42"
    },
    {
      "Original": "NOTE: https://github.com/libexif/libexif/commit/5ae5973bed1947f4d447dc80b76d5cefadd90133",
      "Line": 231083,
      "Type": "NOTE",
      "Description": "https://github.com/libexif/libexif/commit/5ae5973bed1947f4d447dc80b76d5cefadd90133"
    }
  ]
}